binr/rax2/rax2.c

```diff
@@ -35,7 +35,7 @@ static int format_output(char mode, const char *s) {
 		break;
 	case '0': {
 		int len = strlen (s);
-		if (len > 0 && s[len - 1] == 'f') {
+		if (len > 0 && s[len - 1] == 'f' && mode != '0') {
```

Comment:
```c
switch (mode) {
...
case '0': {
...
if (... && ... && mode != '0') {
```
`mode` will never be `'0'` at that `if`.

Reply:
Sorry ... `mode` will always be `'0'` at that `if`.

Reply:
Have to figure out why the test is failing first though.

Reply:
Hmmm `!rax2 121.875f` works but no time to check further ... sorry

Reply:
That's an r2r PR ... make sure its branch has the name `rax2_hex_parsed_as_float` so that Travis here can automatically pick it up.